hoomd/md/pytest/test_methods.py

```diff
@@ -274,6 +274,33 @@ def test_npt_attributes_attached_3d(simulation_factory,
     assert npt.gamma == 2.0
 
 
+def test_npt_thermalize_extra_dof(simulation_factory,
+                                  two_particle_snapshot_factory):
+    """Tests that NPT.thermalize_extra_dof can be called."""
+    all_ = hoomd.filter.All()
+    constant_t = hoomd.variant.Constant(2.0)
+    constant_s = [
+        hoomd.variant.Constant(1.0),
+        hoomd.variant.Constant(2.0),
+        hoomd.variant.Constant(3.0),
+        hoomd.variant.Constant(0.125),
+        hoomd.variant.Constant(.25),
+        hoomd.variant.Constant(.5),
+    ]
+    npt = hoomd.md.methods.NPT(filter=all_,
+                               kT=constant_t,
+                               tau=2.0,
+                               S=constant_s,
+                               tauS=2.0,
+                               couple='xyz')
+
+    sim = simulation_factory(two_particle_snapshot_factory())
+    sim.operations.integrator = hoomd.md.Integrator(0.005, methods=[npt])
+    sim.operations.schedule()
+
+    npt.thermalize_extra_dof(100)
```

Comment:
There isn't an easy way to do this, but it would be nice to check for actually randomization of the extra DOF.

Reply:
I will add the extra DOF to the param dict and then we can validate they get set in this test.